.gitignore

```diff
@@ -37,3 +37,6 @@ public/main.css.map
 
 # Mac
 .DS_Store
+
+# Visual Studio Code
+.vscode
```

Comment:
Add .pyc as well

Reply:
`pyc` files are excluded as well in the same `.gitignore` file